### PR TITLE
feat(searchbar): add autocapitalize as an input attribute on ion-searchbar

### PR DESCRIPTION
**Pull request by kelvindart**
#### Short description of what this resolves:
There is currently not the ability to set the `autocapitalize` attribute on a searchbar, in order to pass it to the native `input` element. With this pull request, you now can.

#### Changes proposed in this pull request:

- The ability to set `autocapitalize`.
- Have this set the property on the native `input`.
- Document the expected values (`none`, `sentences`, `words` and `characters`. `on` and `off` are deprecated as of iOS 5)

**Ionic Version**: 3.6.0

**Fixes**: #12535

### CODE REVIEW

**Comment by drewctate** [comment]
I ran into this, too.  It would be great to get it resolved.

File: src/components/searchbar/searchbar.ts [hunk 1]
```diff
@@ -116,6 +117,11 @@ export class Searchbar extends BaseInput<string> {
   }
 
   /**
+   * @input {string} Set the input's autocapitalize property. Values: `"none"`, `"sentences"`, `"words"`, `"characters"`. Default `"sentences"`.
+   */
+  @Input() autocapitalize: string = 'sentences';
```
Comment (manucorporat) [inline src/components/searchbar/searchbar.ts]: @kelvindart why sentences by default?

Comment (kelvindart) [reply src/components/searchbar/searchbar.ts]: This is the default behaviour, so I didn't want to introduce a breaking change (see: https://developer.apple.com/library/content/documentation/AppleApplications/Reference/SafariHTMLRef/Articles/Attributes.html#//apple_ref/doc/uid/TP40008058-autocapitalize).

> If the autocapitalize attribute is present but no value is specified (as in `<input autocapitalize>`), the default value is sentences. If the autocapitalize attribute is not present, the default value is sentences for form and none for input elements with type set to password as in `<input type="password">`. For all other form controls, the default value is inherited from the enclosing form element.

**Comment by Ionitron** [comment]
Hello and thank you for contributing to Ionic! We have been working on porting all of the Ionic components to web components and have recently updated `master` to reflect this. This significant change has caused this pull request to break. While we really appreciate the time and effort you put into creating this, we are not able to merge it because of the newly introduced conflicts. We are extremely sorry about this. We will not be merging any more features in to v3. If this is a feature and you have the time, please resubmit this PR against the `master` branch. If this is a critical security issue in v3, we would greatly appreciate it if you would resubmit the PR against the new `v3` branch. Thanks so much for your time!

**Pull request closed without merging**